src/lib/with-layout.js

```diff
@@ -84,14 +84,18 @@ export default function withLayout (Component) {
       this.setState({ toggleShowClass: !this.state.toggleShowClass })
     }
 
-    toggleMiniBag() {
-      this.setState({ minibagDisplayed: !this.state.minibagDisplayed })
+    toggleMiniBag(displayed) {
+      this.props.dispatch(toggleMiniBag(displayed))
```

Comment:
This is updated so that minibag state (whether it is shown or not) is only kept in one place, i.e. Redux.